lib/KSelect/index.vue

```diff
@@ -214,14 +220,14 @@
         },
       },
       /**
-       * Placeholder
+       * Sets the placeholder value
        */
       placeholder: {
         type: String,
         default: '',
       },
       /**
-       * Label
+       * Set default label
```

Comment:
`* Specifies the label text. Default is  empty  if not provided.`

Reply:
Thank you @AllanOXDi for reviewing PR.